Source/Tools/LinuxEmulation/LinuxSyscalls/FileManagement.cpp

```diff
@@ -503,6 +503,21 @@ std::pair<int, const char*> FileManager::GetEmulatedFDPath(int dirfd, const char
   return std::make_pair(RootFSFD, &SubPath[1]);
 }
 
+bool FileManager::IsSelf(const char* Pathname) const {
+  if (!Pathname) {
+    return false;
+  }
+
+  char PidSelfPath[50];
+  snprintf(PidSelfPath, 50, "/proc/%i/exe", CurrentPID);
```

Comment:
s/50/sizeof(PidSelfPath)/

Reply:
(Feel free to ignore this one if you think it's silly. It's a bit silly.)

Reply:
Did it.